### DISCUSSION TITLE
Preliminary Lore Dump

### DISCUSSION THREAD

**Discussion by Nodding** [fornclake/TetraForce#294] [Ideas]
**Dumping this for archival reasons, in no way definitive.**

A lot of adventure game titles typically have you save a princess and/or the world by defeating a villain. These very brief “paragraphs” have a goal to propose a more indepth villain, with a somewhat compelling motive. 
We established briefly that the items found in dungeons were originally ours to begin with, and the players get them back. This means in universe, the players are established heroes, who have probably been hired before for other heroic duties. There should be an established kingdom our heroes operate in. There should be an easily identifiable “villain group,” the thieves who steal small things (including the heroes’ items.) 
The King, referred to as the Eternal Emperor, has been in power for as long as the people of the kingdom have known, as well as the Eternal Empress, his wife. The kingdom is worried as the Empress has fallen ill, and the Eternal Emperor hires the hero to go and collect (INSERT NUMBER OF DUNGEONS) ingredients for a potion to cure her.
The ingredients are varied, starting as crystals/rocks, but each seems to come from more organic origins (leaves, fossils, etc.) There should be some hints to parts of old empty houses/caves that warn of the actions of the Emperor/Empress being mysterious, mainly always having a Great War every couple hundreds of years that ends up getting the country massacred. Barely any citizens survive the wars with countries no one knows about until the war is over. Maps around the country never make sense and always have conflicting borders. It is as if the wars are with fake countries, used as a scapegoat for the disappearance of the country.
Once all ingredients are collected, the heroes give them to the king. He then tells the heroes to leave the country. The final ingredient has not been collected yet. It is the collective will of the entire country. The Emperor and Empress have been alive for thousands of years by recharging their lives with the lives of their country. The Empress’s power has faded and she is again aging, hence why she needs to recharge. The Emperor, who loves his wife, is willing to sacrifice everyone to continue to be with his wife. He plans to activate the machine that will “volunteer” the country’s will. The Emperor is gracious of the heroes’ actions and will allow them to leave, unharmed. We could have a joke “bad” ending where you take the offer up. Hopefully they won't, and will storm the underground fortress/laboratory of the Emperor, to shut down the machine and stop them from sacrificing the innocent like this again.

**Comment by yeslayla** [comment]
I like it. Maybe the shrine at the beginning there could be the empress laying on a ritual stone with the emperor nearby?

Since the game is multiplayer, maybe you could be from an adventurers guild? Maybe it could be a place in town that could eventually give randomly generated kill quests in turn for Tetrans.

**Reply by Nodding** [reply]
I love it lol. In terms of demo I think the emperor there to give a call to action with his wife there would be good for it.
